use std::error::Error;

use advent_of_code::aoc2020;

fn main() -> Result<(), Box<dyn Error>> {
    aoc2020::run()?;

    Ok(())
}